release/sdk-config.yaml.tmpl

```diff
@@ -18,7 +18,11 @@ commands:
 - name: build
   path: damlc/damlc
   args: ["build", "--project-check"]
-  desc: "Build the current DAML project into a DAR file"
+  desc: "Build the DAML project into a DAR file"
+- name: deploy
+  path: daml-helper/daml-helper
+  desc: "Build the DAML project and deploy to sandbox"
```

Comment:
I think we should leave the `desc` field off for now. This will keep the command hidden until we are ready to release it to public. See the `ide` command below for example, which is also hidden.

Reply:
ok, done